pkg/execute/executor.go

```diff
@@ -217,9 +225,6 @@ func runKubectlCommand(args []string, clusterName string, isAuthChannel bool) st
 		}
 		finalArgs = append(finalArgs, arg)
 	}
-	if isAuthChannel == false {
-		return ""
-	}
```

Comment:
Please verify if `BotKube notifier` and `BotKube filters` commands are working only from configured channel regardless of `restrictAccess` flag

Reply:
I tested the flow and its working now.